api/translations/validators.de.yml

```
@@ -0,0 +1,5 @@
+Either this value or {{ other }} should not be null.: "Entweder dieser Wert oder {{ other }} müssen nicht null sein"
```

Tönt etwas holprig. Vorschlag:
```suggestion
Either this value or {{ other }} should not be null.: "Dieser Wert und {{ other }} dürfen nicht beide null sein."
```

---

done, see the diff here: https://github.com/ecamp/ecamp3/compare/0fe9d6079a80c42a07f3670e3b49ffdff8d72d63..2ad763ae2cd2528bbfc798782f5bf62a3e476208

---

So ist es aber ungenau. Es dürfen nicht beide `null` sein, aber es dürfen auch nicht beide ausgefüllt sein.

---

True, `AssertEitherIsNull` validates both cases, but works with 2 different validation messages.
```php
    public string $messageBothNull = 'Either this value or {{ other }} should not be null.';
    public string $messageNoneNull = 'Either this value or {{ other }} should be null.';
```

The first one I'd still translate as above.

The second one is actually not translated at the moment. I'd propose "Entweder dieser Wert oder {{ other }} muss null sein."

---

I propose we add the translations themselves in a later PR.